man/systemd.xml

```diff
@@ -940,10 +940,14 @@
       <varlistentry>
         <term><varname>systemd.confirm_spawn=</varname></term>
 
-        <listitem><para>Takes a boolean argument. If
-        <option>yes</option>, the system manager (PID 1) asks for
-        confirmation when spawning processes. Defaults to
-        <option>no</option>.</para></listitem>
+        <listitem><para>Takes a boolean argument or a path to the
+        virtual console where the confirmation messages should be
+        emitted. If <option>yes</option>, the system manager (PID 1)
+        asks for confirmation when spawning processes and use by
```

Comment:
"ask for confirmation when spawning processes using `<option>/dev/console</option>`".